vembrane/ann_types.py

```diff
@@ -289,14 +324,12 @@ def convert(self, key: str, value: str) -> Tuple[str, AnnotationType]:
         int,
         description="The relative position of the variation in the aligned TFBP",
     ),
-    # TODO parse flag:
-    #  "a flag indicating if the variant falls in a high information position of a
-    #  transcription factor binding profile (TFBP)"
-    # "HIGH_INF_POS": AnnotationEntry(
-    #     "HIGH_INF_POS",
-    #     description="A flag indicating if the variant falls in a high information "
-    #                 "position of a transcription factor binding profile (TFBP)",
-    # ),
+    "HIGH_INF_POS": AnnotationEntry(
+        "HIGH_INF_POS",
+        typefunc=lambda x: True if x == "Y" else False,
```

Comment:
```suggestion
        typefunc=lambda x: if x == "Y",
```
Though, if you think `True if ... else False` is more readable, I'm happy to leave it as is!
If micro-optimization performance is of interest,
```suggestion
        typefunc="Y".__eq__,
```
;).